llvm/test/Transforms/LoopVectorize/AArch64/fully-unrolled-cost.ll

```diff
@@ -31,8 +31,8 @@ for.body:                                         ; preds = %entry, %for.body
   %arrayidx2 = getelementptr inbounds i8, ptr %b, i64 %i.iv
   %1 = load i8, ptr %arrayidx2, align 1
   %conv3 = zext i8 %1 to i64
-  %mul = mul nuw nsw i64 %conv3, %conv
-  %add = add i64 %mul, %sum
+  %div = udiv i64 %conv3, %conv
```

Comment:
why are those changes needed?

Reply:
Would be good to keep the original version as well (possibly as part of one of the `partial-reduce-...` tests) to make sure we have test coverage for the case where the cost models disagree?

Reply:
That sounds like a good idea, done.